s2i/s2i.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: build.knative.dev/v1alpha1
+kind: BuildTemplate
+metadata:
+  name: s2i
+spec:
+  parameters:
+  - name: BUILDER_IMAGE
+    description: The location of the buildah builder image.
```

Comment:
No wait a sec, this `BUILDER_IMAGE` isn't `BUILDAH_BUILDER_IMAGE` so this `description` should be e.g. something like _"The location of the image containing the s2i tool.",_ according to the README?

Reply:
OK now